vision/MANIFEST.in

```diff
@@ -1,4 +1,4 @@
-include README.rst LICENSE
+include README.rst LICENSE requirements.txt
 recursive-include google *.json *.proto
```

Comment:
No `.proto` file included, AFAICT.  I think it should be shipped as part of any library generated from it.